drivers/volume/portworx/portworx.go

```diff
@@ -246,27 +280,54 @@ func (p *portworx) initPortworxClients() error {
 	logrus.Infof("Using %v:%v as endpoint for portworx gRPC endpoint", endpoint, p.sdkPort)
 
 	// Setup REST clients
-
-	endpointPort := fmt.Sprintf("http://%v:%v", endpoint, p.restPort)
-	clnt, err := clusterclient.NewClusterClient(endpointPort, "v1")
+	if isTLSEnabled() {
+		scheme = "https"
+	} else {
+		scheme = "http"
+	}
+	p.endpoint = fmt.Sprintf("%s://%v:%v", scheme, endpoint, p.restPort)
+	logrus.Infof("Using %s as the endpoint", p.endpoint)
+	clnt, err := clusterclient.NewClusterClient(p.endpoint, "v1")
 	if err != nil {
 		return err
 	}
 	p.clusterManager = clusterclient.ClusterManager(clnt)
 
-	clnt, err = volumeclient.NewDriverClient(endpointPort, "pxd", "", "stork")
+	p.stopChannel = make(chan struct{})
```

Comment:
stopChannel is already being init'd in Init(). I think you added these by mistake during a merge